package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
Sounds good. If the goal is to make tests runnable after a fresh `git clone`, then we should probably install the `develop` branch of half penny, instead of the latest NPM version. This is because it is the `develop` branch of _nodeapi_ that gets checked out by default. If we were to install the current production version of halfpenny into the develop branch, the tests would likely fail anyway...

Thoughts?